app/models/application_reference.rb

```diff
@@ -49,6 +49,18 @@ def self.not_failed
     where.not(feedback_status: %i[feedback_refused cancelled cancelled_at_end_of_cycle])
   end
 
+  def self.referees_to_chase(chase_referee_by:, rejected_chased_ids:)
+    joins(:application_form)
+      .feedback_requested
+      .where(
+        application_forms: {
+          recruitment_cycle_year: ApplicationForm.select('candidate_id').maximum(:recruitment_cycle_year),
```

Comment:
@steventux I move the existing code to this method but now I realise that might be a bug in this old code. Tomorrow (when appply 1 deadline has passed) there will be 2023 application forms recruitment cycle when people carry over their application and if this criteria exists we will only nudge 2023 references (which will be requested only in October). I do wonder if we should use `RecruitmentCycle.current_year`? Wdyt?
I hope I understood right otherwise ignore me.

Reply:
Done! After adding a test, this line was resulting in a bug in the old implementation where if our current time is between apply 1 deadline and find closes we should chase references on apply again. Fixed and pushed ♻️